packages/api/src/controllers/RateController.js

```diff
@@ -0,0 +1,28 @@
+const Kucoin = require('../kucoin')
+const Coinbase = require('../coinbase')
+
+class RateController {
+  async getUSDRate (request, response) {
+    const kucoinResponse = await Kucoin.getUSDRate().catch((err) => {
```

Comment:
I think we can move try catch down in the kucoin and just return null in case of errors